include/core/surface_tension_model.h

```diff
@@ -109,4 +136,102 @@ class SurfaceTensionConstant : public SurfaceTensionModel
   const double surface_tension_coefficient;
 };
 
+/**
+ * @brief Linear surface tension. The surface tension is given by:
+ * sigma_0 + dsimga/dT * T.
+ */
+class SurfaceTensionLinear : public SurfaceTensionModel
```

Comment:
The only comment I have about this is that the model currently implemented is a linear variation only with temperature, but that is not what's reflected in the name of the class. Maybe we could rename it or make it more general (linear variation for any field). 

However, I don't know if it is only a detail since it doesn't affect the implementation at all :o

Reply:
Since the dependency to fields is enabled in the constructor, I was think of renaming it when another linear model will be implemented and leave it like this for now. What do you think?

Reply:
Seems like a legit argument to me, I'm ok with this formulation (although I had the same thought)